views/index.ejs

```diff
@@ -93,11 +93,16 @@
         <h2>Average % Increase</h2>
         <% let sumChange = 0 %>
         <% for (let i=0; i < info.length; i++) {%>
-            <% let prev = parseInt(info[i].prevIncome, 10)%>
-            <% let tech = parseInt(info[i].techIncome, 10)%>
-            <% sumChange += ((tech - prev)/prev * 100) %>
-            <%}%>
-            <h3 class="text-success"><%= (sumChange/(info.length)).toFixed(2) %>%</h3>
+          <% let prev = parseInt(info[i].prevIncome, 10)%>
+          <% let tech = parseInt(info[i].techIncome, 10)%>
+          <% sumChange += ((tech - prev)/prev * 100) %>
+        <%}%>
+          
+        <% if (sumChange) { %>
+          <h3 class="text-success"><%= (sumChange/(info.length)).toFixed(2) %>%</h3>
+        <%} else { %>
+          <h3 class="text-success">No data yet</h3>
+        <%}%>
```

Comment:
This change prevents the initial state of `NaN%` and instead prints `No data yet`.